performance_test/scripts/fio-cmp.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: kbench-pvc-1
+spec:
+  volumeMode: Filesystem
+  #volumeMode: Block
+  #storageClassName: longhorn  # replace with your storage class
+  storageClassName: local-path
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 55Gi
```

Comment:
What's the memory size in each node?
Typically, we use 5x of memory size as the storage size, because we hope the io sometimes from on-disk data and sometime from memory in buffer io test.

Reply:
the default ec2 instance used is c5d.2xlarge, it has 16GB of memory

Reply:
Then we can update the storage to `80Gi`.

Reply:
[test size must be 10% smaller than the PVC size due to filesystem also took space](https://github.com/yasker/kbench/blob/main/deploy/fio.yaml), so maybe test size = 80Gi, storage size ~= 89Gi ?

Reply:
Yeah. But this is a rough value. 4x-5x is okay.

Reply:
Can we make this configurable via Jenkins job?